salt/utils/win_pwsh.py

```diff
@@ -0,0 +1,67 @@
+import salt.modules.cmdmod
+import salt.utils.json
+import salt.utils.platform
+from salt.exceptions import CommandExecutionError
+
+__virtualname__ = "win_pwsh"
+
+
+def __virtual__():
+    """
+    Only load if windows
+    """
+    if not salt.utils.platform.is_windows():
+        return False, "This utility will only run on Windows"
+
+    return __virtualname__
```

Comment:
```suggestion
__virtualname__ = "win_pwsh"


def __virtual__():
    """
    Only load if windows
    """
    if not salt.utils.platform.is_windows():
        return False, "This utility will only run on Windows"

    return __virtualname__
```
Please don't. Don't add anything else to `__utils__`.

Reply:
I removed it for both `win_pwsh` and `win_lgpo_netsh`